srcpkgs/python3-cached-property/patches/p267.patch

```diff
@@ -0,0 +1,101 @@
+diff --git a/cached_property.py b/cached_property.py
```

Comment:
Can you link source for this patch? Or at least explain why it's necessary?

Reply:
source appears to be https://github.com/pydanny/cached-property/pull/267, which is irrelevant for now